doc-content/enterprise-only/installation/clustering-quartz-proc.adoc

```diff
@@ -0,0 +1,103 @@
+[id='clustering-quartz-proc_{context}']
+= Configuring Quartz timer service
+
+When you run {KIE_SERVER} in a cluster you must configure the Quartz timer service.
+
+Before you configure a database on your application server, you must prepare the database for Quartz to create Quartz tables, which will hold the timer data, and the Quartz definition file.
+
+.Prerequisites
+
+* A supported non-JTA data source is available. For example, a PostgreSQL data source.
+
+* The database must be connected to your application server.
```

Comment:
What database? Is this the data source in the preceding prereq? If so , I would roll this up with the previous:
A supported non-JTA data source is connected to your application server, for example a PostgreSQL data source.

Reply:
Done